lib/nfg_ui/components/utilities/disable_withable.rb

```diff
@@ -14,7 +14,7 @@ def disable_with
         private
 
         def default_disable_with
-          @default_disable_with ||= view_context.ui.nfg(:icon, :loader, text: 'Saving...')
+          @default_disable_with ||= view_context.ui.nfg(:icon, :loader, text: 'Saving')
```

Comment:
@farhajawed if we're documenting the disable_with functionality of buttons, we've removed the 3 dots / ellipsis

Reply:
Here: https://github.com/network-for-good/nfg_ui_display_app/pull/1/files#r669038018